lib/install/validate/config_to_data.go

```diff
@@ -296,22 +297,26 @@ func setHTTPProxies(d *data.Data, conf *config.VirtualContainerHostConfigSpec) e
 		return nil
 	}
 	for _, env := range persona.Cmd.Env {
-		if !strings.HasPrefix(env, httpProxy) && !strings.HasPrefix(env, httpsProxy) {
+		if !strings.HasPrefix(env, httpProxy) && !strings.HasPrefix(env, httpsProxy) && !strings.HasPrefix(env, noProxy) {
```

Comment:
This is likely much neater if using switch - something like:
```go
for _, env := range persona.Cmd.Env {
  val := strings.Split(env, "=")
  switch {
    case strings.HasPrefix(env, httpProxy):
      ...
    case strings.HasPrefix(env, httpsProxy):
      ...
    case strings.HasPrefix(env, noProxy):
      ...
}
```

Reply:
Good suggestion. Just refactored to use switch.